docs/src/content/docs/basic-concepts.mdx

```diff
@@ -110,32 +110,27 @@ Components are executed using `gsx.execute()`, which processes the JSX tree from
 const result = await gsx.execute(<MyComponent input="some data" />);
 ```
 
-Components can be tested and evaluated in isolation, making it easy to debug and verify individual steps of your workflow. This is particularly valuable when building complex LLM applications that need robust evaluation.
-
-Rather than having to run an entire workflow to test a change to a single component, you can test just that component, dramatically speeding up your dev loop. This isolation also makes unit testing more manageable, as you can create specific test cases without having to worry about the rest of the workflow.
-
-## Providers
-
-Providers are special components that manage configuration and dependencies for your workflow. They're similar to React's Context providers - they just pass configuration down to child components.
-
-The main provider available today is the `OpenAIProvider`, which manages your OpenAI API key:
+You'll often use `gsx.execute()` to run a top level workflow as shown above, but you can also use it to run a sub-workflow within a component. This allows you to work with a component in TypeScript without having to drop into the JSX layer. This is especially useful for more complex operations like map/reduce:
```

Comment:
should also mention here that `gsx.execute` within a component maintains all current context and parent/child relationships with the current component and the rest of the workflow.

Reply:
added